packages/main/src/ListItem.ts

```diff
@@ -481,6 +482,7 @@ abstract class ListItem extends ListItemBase {
 			ariaHaspopup: this.accessibilityAttributes.hasPopup,
 			setsize: this.accessibilityAttributes.ariaSetsize,
 			posinset: this.accessibilityAttributes.ariaPosinset,
+			listItemAriaLabel: this.typeActive ? ListItem.i18nBundle.getText(LIST_ITEM_ACTIVE) : undefined,
```

Comment:
Not sure this is the correct place because if I use accessibleName, it will be read before item's label.

As I saw how it is used in the code it is used to to define post label for the tree item.